asv_bench/benchmarks/io/parquet.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import modin.pandas as pd
+
+from ..utils import (
+    ASV_USE_IMPL,
+    IMPL,
+    execute,
+    get_shape_id,
+    prepare_io_data_parquet,
+    get_benchmark_shapes,
+)
+
+
+class BaseReadParquet:
```

Comment:
I don't think we should add a base class until we have multiple subclasses.

Reply:
We do this with read_csv, and we may add other things for parquet longer term so just wanted consistency.

Reply:
We do this with read_csv, so wanted some consistency in case in future we add things for parquet.

Reply:
I generally prefer in cases like this to assume [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) and write the simpler code. However, I won't insist.

Reply:
I agree with @mvashishtha, let's keep it simple.